terraform/physical/util/create-s3-batch.sh

```diff
@@ -5,12 +5,18 @@ set -euo pipefail
 AWS_LOGGING_BUCKET="$1"
 AWS_SOURCE_BUCKET="$2"
 AWS_REPLICATION_ROLE="$3"
-AWS_PROFILE="$4"
-AWS_ACCOUNT="$5"
+AWS_ACCOUNT="$4"
+AWS_PROFILE="$5"
```

Comment:
This isn't a problem, more of a curiosity: Why the position change here?

Reply:
Because the profile selection is optional so it needs to go last.